oak/server/rust/oak_runtime/src/lib.rs

```diff
@@ -14,6 +14,15 @@
 // limitations under the License.
 //
 
+//! Oak Runtime implementation
+//!
+//! # Features
+//!
+//! The `oak_debug` feature enables various debugging features, including
+//! data structure introspection functionality.  This feature should only
```

Comment:
```suggestion
//! data structure introspection functionality. This feature should only
```

It seems most linters / formatters remove double spaces in other contexts, though not in Rust comments, but worth trying to be consistent, if possible. Not big deal anyways.

Reply:
Done (but decades of muscle memory means that double spaces are likely to creep in anywhere that's not auto-checked!)